libjsonutils/cmake/FindRapidJSON.cmake

```diff
@@ -1,46 +1,55 @@
 #FindRapidJSON.cmake
-# 
+#
 # Finds the rapidjson library
 #
+# from http://rapidjson.org/
+#
 # This will define the following variables
 #
-#    RapidJSON_FOUND
-#    RapidJSON_INCLUDE_DIRS
+#    RAPIDJSON_FOUND
+#    RAPIDJSON_VERSION
+#    RAPIDJSON_INCLUDE_DIRS
 #
 # and the following imported targets
 #
 #     RapidJSON::RapidJSON
 #
 # Author: Pablo Arias - pabloariasal@gmail.com
+#
+# clausklein$ cmakelint *.cmake
+# FindRapidJSON.cmake:0: Find modules should use uppercase names;
+#   consider using FindRAPIDJSON.cmake [convention/filename]
+# Total Errors: 1
 
-find_package(PkgConfig)
+find_package(PkgConfig REQUIRED)
 pkg_check_modules(PC_RapidJSON QUIET RapidJSON)
 
-find_path(RapidJSON_INCLUDE_DIR
+find_path(RAPIDJSON_INCLUDE_DIR
     NAMES rapidjson.h
     PATHS ${PC_RapidJSON_INCLUDE_DIRS}
     PATH_SUFFIXES rapidjson
 )
 
-set(RapidJSON_VERSION ${PC_RapidJSON_VERSION})
+set(RAPIDJSON_VERSION ${PC_RapidJSON_VERSION})
 
-mark_as_advanced(RapidJSON_FOUND RapidJSON_INCLUDE_DIR RapidJSON_VERSION)
+# NOTE: (cmakelint) All vars should be uppercase! CK
+mark_as_advanced(RAPIDJSON_FOUND RAPIDJSON_INCLUDE_DIR RAPIDJSON_VERSION)
```

Comment:
Uhh, no. That hasn't been the case in a long time. Variables are better matched to the module name.